queries.sql

```diff
@@ -15,3 +15,50 @@ select * from animals where neutered = true
 select * from animals where name not like '%Gabumon'
 
 select * from animals where weight_kg between  10.4 and 17.3
+
+vetclinic=# BEGIN
+vetclinic-# ;
+BEGIN
+vetclinic=*# UPDATE ANIMALS SET SPECIES = 'UNSPECIFIED' WHERE SPECIES IS NULL
+vetclinic-*# ;
+UPDATE 11
+vetclinic=*# rollback;
+ROLLBACK
+vetclinic=# update animals set species = 'digimon' where name like '%mon';
+UPDATE 6
+vetclinic=# update animals set species = 'pokemon' where species is null;
+UPDATE 5
+COMMIT;
+vetclinic=# BEGIN;
+BEGIN
+vetclinic=*# DELETE FROM ANIMALS;
+DELETE 11
+vetclinic=*# ROLLBACK;
+ROLLBACK
+vetclinic=# delete from animals where date_of_birth > '2022-01-01';
+DELETE 1
+vetclinic=# BEGIN;
+BEGIN
+vetclinic=*# SAVEPOINT DEL_ANIMAL_JAN22;
+SAVEPOINT
+vetclinic=*# update animals set weight_kg = weight_kg * -1;
+UPDATE 10
+vetclinic=*# rollback to DEL_ANIMAL_JAN22;
+ROLLBACK
+vetclinic=*# update animals set weight_kg = weight_kg * -1 where weight_kg < 0;
+UPDATE 0
+vetclinic=*# commit;
+COMMIT
+
+select count(*) as total_animals from animals 
+
+select count(*) as not_escaped from animals where escape_attempts = 0 
+
+select avg(weight_kg) as Average_weight from animals 
+
+select name, escape_attempts from animals where escape_attempts =(select max(escape_attempts) from animals);
+
```

Comment:
- [ ] This query supposes to answer Who escapes the most, neutered or not neutered animals? not Who is the animal escapes the most. So please fix that.
Hint: It will be something like that:
```sql
SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
```

Reply:
done, as advised